### ISSUE TITLE
Color processing

### ISSUE THREAD

**Issue by kubiqsk** [scssphp/scssphp#459]
Hi,

I just tried to compile new Bootstrap 5.1.1 and I can see many differences.

I understand, that you don't handle prefixes, but it would be great if you can provide some "filter" or some array that we can manually specify and it will prefix that values according it.

But the real problem I want to write about is about completely different colors, like:

Original BS compiled file:

```
.table-info {
  --bs-table-bg: #cff4fc;
  --bs-table-striped-bg: #c5e8ef;
  --bs-table-striped-color: #000;
  --bs-table-active-bg: #badce3;
  --bs-table-active-color: #000;
  --bs-table-hover-bg: #bfe2e9;
  --bs-table-hover-color: #000;
  color: #000;
  border-color: #badce3;
}
```

scssphp:

```
.table-info {
  --bs-table-bg: #cff4fc;
  --bs-table-striped-bg: #d1f5fc;
  --bs-table-striped-color: #000;
  --bs-table-active-bg: #d4f5fc;
  --bs-table-active-color: #000;
  --bs-table-hover-bg: #d3f5fc;
  --bs-table-hover-color: #000;
  color: #fff;
  border-color: #d4f5fc;
}
```

or another issue

Original BS compiled file:

```
.text-primary {
  --bs-text-opacity: 1;
  color: rgba(var(--bs-primary-rgb), var(--bs-text-opacity)) !important;
}
```

scssphp:

```
.text-primary {
  --bs-text-opacity: 1;
  color: rgba(var(--bs-primary-rgb)) !important;
}
```

**Comment by stof** [comment]
> I understand, that you don't handle prefixes, but it would be great if you can provide some "filter" or some array that we can manually specify and it will prefix that values according it.

dart-sass does not perform any prefixing either. The Bootstrap build process runs autoprefixer on the *output* of the Sass compilation. So this is clearly out of scope of the project.

Regarding color issues, which version of scssphp are you using ? It is a known issue that compiling Bootstrap 5.0.2+ with scssphp <1.6.0 is broken (the new way they perform divisions to avoid the /-as-division deprecation revealed a bug in our implementation). So make sure you use scssphp 1.6.0.

**Comment by kubiqsk** [comment]
yes, i have newest version, downloaded just 1-2 days before I created this ticket
I compiled bootstrap.scss like this:
```
require_once 'scssphp-1.6.0/scss.inc.php';
use ScssPhp\ScssPhp\Compiler;
$scss = new Compiler();
$scss->setImportPaths('bootstrap-5.1.1/scss');
$result = $scss->compileString('@import "bootstrap.scss";');
file_put_contents( 'style.css', $result->getCss() );```

**Comment by stof** [comment]
@kubiqsk should be fixed in the latest 1.x branch (that will be released as part of 1.7.0 in the next few days)
